rails/app/policies/user_policy.rb

```diff
@@ -11,7 +11,7 @@ def index?
   end
 
   def edit?
-    current_user.admin? || (
+    current_user.super_admin || current_user.admin? || (
```

Comment:
should this be `super_admin?` (I think the predicate method comes from AR enums)
```
pry(main)> User.last.method(:admin?).source_location
  User Load (11.5ms)  SELECT "users".* FROM "users" ORDER BY "users"."id" DESC LIMIT $1  [["LIMIT", 1]]
=> ["/usr/local/bundle/ruby/2.5.0/gems/activerecord-6.1.7/lib/active_record/enum.rb", 235]
```

What's weird is that there is a `super_admin` boolean on user too, but at least in my seed file I end up with this:

```
[12] pry(main)> User.last
  User Load (2.6ms)  SELECT "users".* FROM "users" ORDER BY "users"."id" DESC LIMIT $1  [["LIMIT", 1]]
=> #<User id: 6, email: "public@terrastories.io", created_at: "2023-02-03 03:05:08.225443000 +0000", updated_at: "2023-02-03 03:05:08.225443000 +0000", role: "super_admin", community_id: nil, super_admin: false, username: "terrastories-public", name: nil>
[13] pry(main)> User.last.super_admin
  User Load (0.9ms)  SELECT "users".* FROM "users" ORDER BY "users"."id" DESC LIMIT $1  [["LIMIT", 1]]
=> false
[14] pry(main)> User.last.super_admin?
  User Load (0.6ms)  SELECT "users".* FROM "users" ORDER BY "users"."id" DESC LIMIT $1  [["LIMIT", 1]]
=> true
```

Reply:
You're right! Updated.